packages/core/src/editor/editor.css

```diff
@@ -156,7 +156,6 @@ Tippy popups that are appended to document.body directly
 
 .bn-editor [data-content-type="table"] th {
   font-weight: bold;
-  text-align: left;
```

Comment:
Looks like this still needs to be set as table headers default to center alignment, so they're still centered even when the text alignment is set to left

Reply:
Maybe this is a style thing, but table headers are usually centered.

Also, I didn't want this CSS to conflict with manually setting the text alignment on the cell level given it is a low specificity selector: 

```css
[data-text-alignment="right"] {
  justify-content: flex-end;
  text-align: right;
}
```

Reply:
Makes sense, though I think it's still weird that currently changing text alignment on a header cell via the formatting toolbar has no effect between alignment left/center. I think if we want them to be centered by default, we should do it through the text alignment prop.